package.json

```diff
@@ -38,7 +38,7 @@
     "fs-extra": "^1.0.0",
     "jsesc": "^2.2.0",
     "unicode-9.0.0": "^0.7.0",
-    "unicode-tr51": "^7.0.2"
+    "unicode-tr51": "^8.0.0"
```

Comment:
Could you update this to v8.0.1 explicitly please? 8.0.0 is missing `Emoji_Component.js`.